src/modules/fancyzones/editor/FancyZonesEditor/GridEditorWindow.xaml

```diff
@@ -10,7 +10,7 @@
         Width="528"
         SizeToContent="Height"            
         Background="White"
-        ResizeMode="NoResize"
+        ResizeMode="CanMinimize"
```

Comment:
Why is CanMinimize part of this change? I would pull this out if it's not germane to the small screens fix.

Reply:
Apologies for not being more clear. I was referring to this change across all the dialogs. 

There is more of a design discussion behind this, so for now you should remove this change to keep this PR about the sizing.

The reason there is more discussion to have here is that this is viewed as a more atomic experience where we expect the user to start the action and complete it without switching away. Atomic experiences should keep the ui in front of the users to move them along. I understand that moving the dialogs around when they obscure editing controls is not ideal, but I think that is preferred over the dialog minimizing. This, of course, is debatable, which is why I'm saying we should separate this out to move the central change forward.

Thanks

Reply:
Jeff, I rolled back the change.